2022-07_sprint/sprint_parties/index.md

```diff
@@ -7,4 +7,5 @@ organizers
 community_partners
 sponsors
 contributors
+2022_07_media_kit
```

Comment:
```suggestion
media_kit
```

The file is called `media_kit.md` and is is in this same folder. The toctree needs to use relative paths excluding the extension.